tests/cephadm/test_add_host_to_maintenance_using_flags.py

```diff
@@ -0,0 +1,41 @@
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.utils import get_node_by_id
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Create a pool for rbd application that has
+    """
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    print(cephadm)
+    orch_obj = RadosOrchestrator(node=cephadm)
+    pool_config = config.get("pool_config")
+    instance = config.get("node")
+    instance_name = get_node_by_id(ceph_cluster, instance).hostname
+
+    # Create a pool
+    out = orch_obj.create_pool(**pool_config)
```

Comment:
is pool creation required here as we are directly proceeding to host maintenance? Please ignore if I got it wrong

Reply:
Yes Pranav, thats what the reproducer steps mention for this bug.